itl/command-plugins.conf

```diff
@@ -423,8 +427,13 @@ object CheckCommand "http" {
 		}
 		"-C" = {
 			value = "$http_certificate$"
-			description = "Minimum number of days a certificate has to be valid. This parameter explicitely sets the port to 443 and ignores the URL if passed."
+			description = "Minimum number of days a certificate has to be valid. This parameter explicitely sets the port to 443 and ignores the URL if passed"
```

Comment:
The removed `.` produces unnecessary diffs, please revert it.

Reply:
Shouldn't the "description syntax" be unique? The sentences end without a '.' in the other arguments.  
I can remove it, but then it will never be standardized.

Reply:
I don't think this is an "unnecessary diff"...  
No description end with a '.', so it would therefore only be consequent to change that as well.

Reply:
> No description end with a '.'

Regarding the http check command, yes, you are right, but there are plenty of other check commands that do end with a full stop. But to be honest, I don't care that much about this change, it was just unnecessary as it doesn't relate to the actual problem in anyway. So, change it or leave it whatever you like.